### DISCUSSION TITLE
React Native: value attribute exists in 0.72 but not in 0.73

### DISCUSSION THREAD

**Discussion by wilsolutions** [appium/appium#20468] [Q&A]
I have an option element in a RN App (react native 0.72) that works fine, it has a value =  'radio button', fine.
However when I upgrade the App to React Native 0.73 with no changes the "value" attribute is gone.

My workaround was adding a `accessibilityValue={{ text: 'radio button' }}` to the element. 
Is this the way moving forward?
Why the "value" attribute is gone when using RN 0.73?
Thanks

**Comment by KazuCocoa** [comment]
It looks like asking it on the RN side would help. Appium loads contents via OS, so Appium itself cannot inject anything how RN let OS loads them.

**Comment by wilsolutions** [comment]
Hi @KazuCocoa 
Yep, good point, thanks for the heads up.
I will check there.
